src/schemas/cedict.js

```diff
@@ -0,0 +1,36 @@
+const cedict = {
+  db: {
+    name: 'AlpheiosCedict',
+    version: 1,
```

Comment:
This is a version of an IndexedDB database schema (i.e. how many stored do we have and in what format do they store the data).